browser/brave_browser_process_impl.cc

```diff
@@ -155,6 +156,15 @@ BraveBrowserProcessImpl::referrer_whitelist_service() {
   return referrer_whitelist_service_.get();
 }
 
+brave_site_specific::SiteSpecificScriptConfigService*
```

Comment:
The naming seems a bit inconsistent to me. Could we do `brave/components/brave_site_scripts` path and `brave_site_scripts::SiteScriptsService` for the keyed service? That would be more consistent with brave_rewards for instance which is `brave_rewards::RewardsService`

Reply:
or should the component name be `greaselion`?

Reply:
and maybe this should `SiteScriptsDownloadService`?